pages/common/timew.md

```diff
@@ -0,0 +1,19 @@
+# timew
+
+> Time tracker.
+
+- Start a new stopwatch:
+
+`timew {{item_to_time}} start`
```

Comment:
The `{{item_to_time}}` is supposed to be the tag, an identifier for the thing the user is tracking. I didn't realize the man page showed the tag being placed after an initial `timew start`, as both work methods work equally. I've changed it in commit 2b48f56 to match the man page anyway.